Jucer2Reprojucer/main.cpp

```diff
@@ -270,6 +270,51 @@ void writeUserNotes(LineWriter& wLn, const juce::ValueTree& valueTree)
   }
 }
 
+    juce::ValueTree sanitizeConfigurationNames(const juce::String& exporterName, const juce::ValueTree& exporterSettingsWithInvalidConfigNames)
```

Comment:
I know it is not written down explicitly, and there is no CI check for it, but the C++ code of FRUT is formatted using `clang-format`. There is a [`.clang-format` ](https://github.com/McMartin/FRUT/blob/c3dd1775bba0ad0db43389203658151e4db77198/.clang-format) file at the root of the repository, which is currently configured to be used with `clang-format-7.0.0` (you can download binaries for most platforms on http://releases.llvm.org/download.html#7.0.0). I usually run the following:
```
clang-format-7.0.0 -i -style=file -verbose Jucer2Reprojucer/main.cpp
```

Reply:
Yes, sure I can run clang-format on it. I simply didn't notice it. Thanks for pointing it out.

I would suggest to add a check to the CI. [run-clang-format](https://github.com/Sarcasm/run-clang-format) could help with this.

Reply:
> I would suggest to add a check to the CI.

I'll work on that when I have some time 😅

Reply:
Done in https://github.com/McMartin/FRUT/pull/526.